browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -2416,28 +2417,34 @@ TEST_F(KeyringServiceUnitTest, ImportFilecoinAccounts) {
     const char* type;
     const char* public_key;
   } imported_accounts[] = {
-      {"Imported Filecoin account 1",
-       /*"rQG5jnbc+y64fckG+T0EHVwpLBmW9IgAT7U990HXcGk=",*/
-       "ad01b98e76dcfb2eb87dc906f93d041d5c292c1996f488004fb53df741d77069",
-       "t1lqarsh4nkg545ilaoqdsbtj4uofplt6sto26ziy", "secp256k1", ""},
-      {"Imported Filecoin account 2",
-       /*"9zGm0c5xNCYIKKE/QyvumrUDC9vnpaF4GtaOdlr4YQM=",*/
-       "f731a6d1ce7134260828a13f432bee9ab5030bdbe7a5a1781ad68e765af86103",
-       "t1gfpgfwrxdntcmcfqm7epsxrmsxlmsryvjkgat3i", "secp256k1", ""},
-      {"Imported Filecoin account 3",
-       /*"hEZRG5O7HGPT66ufJapik5943MBiyFMACwaLx0aK0TQ=",*/
-       "8446511b93bb1c63d3ebab9f25aa62939f78dcc062c853000b068bc7468ad134",
-       "t1gcfbv323mpexk2pumtkgibtvrulxarnafxryyly", "secp256k1", ""},
-      {"Imported Filecoin account 4",
-       "fbf541635f70a7919efe024235a0d669760938619263c38b8773e398bee91234",
-       "t3wilvmgf76qxkd3aj6inzeilz7gghu5w46zgmu4m5jte752xxgbkjifg2o35w7fky42jzq"
-       "ge7wkrhuobdjg2a",
-       "bls",
-       "b2175618bff42ea1ec09f21b922179f98c7a76dcf64cca719d4cc9feeaf730549414da7"
-       "6fb6f9558e69398189fb2a27a"}};
-  for (size_t i = 0;
-       i < sizeof(imported_accounts) / sizeof(imported_accounts[0]); ++i) {
-    absl::optional<std::string> address;
+    {"Imported Filecoin account 1",
+     /*"rQG5jnbc+y64fckG+T0EHVwpLBmW9IgAT7U990HXcGk=",*/
+     "ad01b98e76dcfb2eb87dc906f93d041d5c292c1996f488004fb53df741d77069",
+     "t1lqarsh4nkg545ilaoqdsbtj4uofplt6sto26ziy", "secp256k1", ""},
+    {"Imported Filecoin account 2",
+     /*"9zGm0c5xNCYIKKE/QyvumrUDC9vnpaF4GtaOdlr4YQM=",*/
+     "f731a6d1ce7134260828a13f432bee9ab5030bdbe7a5a1781ad68e765af86103",
+     "t1gfpgfwrxdntcmcfqm7epsxrmsxlmsryvjkgat3i", "secp256k1", ""},
+    {"Imported Filecoin account 3",
+     /*"hEZRG5O7HGPT66ufJapik5943MBiyFMACwaLx0aK0TQ=",*/
+     "8446511b93bb1c63d3ebab9f25aa62939f78dcc062c853000b068bc7468ad134",
+     "t1gcfbv323mpexk2pumtkgibtvrulxarnafxryyly", "secp256k1", ""}
+#if BUILDFLAG(ENABLE_RUST_BLS)
+    ,
+    {"Imported Filecoin account 4",
+     "fbf541635f70a7919efe024235a0d669760938619263c38b8773e398bee91234",
+     "t3wilvmgf76qxkd3aj6inzeilz7gghu5w46zgmu4m5jte752xxgbkjifg2o35w7fky42jzq"
+     "ge7wkrhuobdjg2a",
+     "bls",
+     "b2175618bff42ea1ec09f21b922179f98c7a76dcf64cca719d4cc9feeaf730549414da7"
+     "6fb6f9558e69398189fb2a27a"}
+#endif
+  };
+  auto* filecoin_keyring =
+      service.GetHDKeyringById(brave_wallet::mojom::kFilecoinKeyringId);
```

Comment:
you shouldn't be getting keyring here
It's move here for lazily created test
https://github.com/brave/brave-core/pull/11866/files#diff-a3ab8c04aa8452906825049c1f5b1bc021578109c9e61b1601d332d51c13bb72L2469

Reply:
done